OWNERS

```diff
@@ -0,0 +1,6 @@
+assignees:
+- migmartri
```

Comment:
It's YAML so I believe it doesn't matter, but I'll change it to match the other repos :+1: